example/src/App.js

```diff
@@ -18,7 +18,7 @@ export default class App extends Component {
       let BNB = new web3.eth.Contract(compiledBNB,'0x763529b9ECE8694442FC7CAA2bEb305D6452B56d');
 
       this.trxRequest = BNB.methods.transfer('0xd396ce692b3c735d9ebd4ed0407c9df17efcee91',web3.utils.toWei('0', 'ether')).send.request({from: "", value: web3.utils.toWei('0', 'ether'), gasPrice: '1'});
-      console.log('trxRequest', this.trxRequest);
+      console.log('trxRequest', this.trxRequest.params[0].to);
```

Comment:
instead of `trxRequest.params[0].to`, set `trxRequest` or `trxRequest.params[0]`